api/v1/backend.go

```diff
@@ -279,3 +286,9 @@ func (in *RestServerSpec) String() string {
 	protocol, url, _ := strings.Cut(in.URL, "://")
 	return fmt.Sprintf("rest:%s://%s:%s@%s", protocol, "$(USER)", "$(PASSWORD)", url)
 }
+
+type BackendOpts struct {
```

Comment:
Actually it's maybe over engineering, I prefer use options property cuz maybe in feature k8up need some options for manage restic or anything.

It's like command optionsis  transfred to yaml. In feature we be able add more options for manage every thing

If you think it's unnecessary, I can change this property and rename to the TLSOptions or use ENV instead of yaml property

Reply:
I'd say generic restic options should belong in a dedicated property outside the `Backend` object. As that would mix up the responsibilities of the `Backend` object.

Reply:
How would you like to desire this property? Which is better, renaming to `TLSOptions` or moving outside the `Backend` object?

Reply:
Just rename it for now.

Reply:
fixed